Automattic/wp-calypso
# VideoPress Onboarding Intent: Add intent modals

## Proposed Changes

This PR adds modals to the VideoPress onboarding intent and the following modal content:
* Video Portfolio
* Jetpack
* Blog

## Testing Instructions

* Apply PR
* Go to /setup/videopress
* Click on "Get a video portfolio"
* ✅ A modal should appear
* ✅ The content should match the selection
* Close the modal
* Click on "Add video to existing site"
* ✅ A modal should appear
* ✅ The content should match the selection
* Close the modal
* Click on "Start a blog with video content"
* ✅ A modal should appear
* ✅ The content should match the selection
* Click on "Create a channel"
* ✅ A modal should appear
* ✅ The content should match the selection
* ✅ The content should be in "coming soon" mode 
* Click on "Upload a video"
* ✅ A modal should appear
* ✅ The content should match the selection
* ✅ The content should be in "coming soon" mode
